galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager/src/main/java/dev/galasa/core/manager/internal/CoreManager.java

```diff
@@ -69,7 +69,7 @@ public void initialise(@NotNull IFramework framework, @NotNull List<IManager> al
 		this.ctf = framework.getConfidentialTextService();
 
 		if(galasaTest.isGherkin()) {
-			GherkinStatements.register(galasaTest, this);
+			GherkinStatements.register(galasaTest, this, allManagers, activeManagers);
```

Comment:
same as http,   does the manager need to determine that it is actually required by examining the statements and then called youarerequried.